### PR TITLE
Set dynamic year in the footer

### LINKED ISSUE #52: Title: Dynamic Year in Copyright Notice
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

The current copyright notice in the project's footer contains a static year (2023). To ensure it remains up-to-date, it would be beneficial to make the year dynamic, automatically updating to the current year. This helps maintain accuracy without manual intervention. Please assign this isssue to me under SWOC 24. Thank you

### Screenshots

_No response_

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

### PR DESCRIPTION
**Pull request by zahidgul107**
# Fixes Issue🛠️

Closes #52 

# Description👨‍💻 

This commit addresses issue #52, which involved making the copyright year in the footer dynamic. I have used jQuery instead of Vanilla Javascript, and the dynamic year is now set using the provided jQuery CDN.

# Type of change📄

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)

# How this has been tested✅

Local host

# Checklist✅ 

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I am an Open Source Contributor

# Screenshots/GIF📷
Screenshot 2024-01-05 010850

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -156,11 +156,12 @@ <h2>Text Animations</h2>
       <p>
         Created By
         <a href="https://github.com/Rakesh9100" target="_blank">Rakesh Roshan ❤️</a>
-        | © 2023: All Rights Reserved
+        | © <span id="year"></span>: All Rights Reserved
       </p>
     </footer>
   </div>
 
+  <script src="https://code.jquery.com/jquery-3.6.4.min.js"></script>
```
[the remark this answers is not in the document]
Comment (zahidgul107) [reply index.html:164]: we can do it pure JavaScript as well, should i use it

[the remark this answers is not in the document]
Comment (Rakesh9100) [reply index.html:164]: Yes @zahidgul107

[the remark this answers is not in the document]
Comment (zahidgul107) [reply index.html:164]: ok thanks @Rakesh9100

**Comment by Rakesh9100** [comment]
Pls add screenshot in the PR description @zahidgul107

**Comment by zahidgul107** [comment]
i have added screenshot in PR Description @Rakesh9100

**Comment by Rakesh9100** [comment]
Good, merged!! Congrats on getting your PR merged to the repo under SWOC S4 🎉 @zahidgul107

**Pull request merged**